Add DisableImplicitFrameworkReferences (#22128)
dotnet/docs: docs/core/project-sdk/msbuild-props.md

</PropertyGroup>
```

### DisableImplicitFrameworkReferences

> The `DisableImplicitFrameworkReferences` property lets you control whether to implicitly reference metapackages, such as [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) or [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library). Metapackages, which are sets of packages that are meaningful together, are implicitly referenced based on the [TargetFramework](#targetframework). Set this property to `true` to disable implicit [PackageReference](#packagereference) items to packages in the metapackage. If you set this property to `true`, you can add explicit references to just the packages you need.

Reviewer: ```suggestion
The `DisableImplicitFrameworkReferences` property lets you control whether to implicitly reference metapackages, such as [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) or [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library). Metapackages, which are packages that consist only of dependencies on other packages, are implicitly referenced based on the [TargetFramework](#targetframework). Set this property to `true` to disable implicit [PackageReference](#packagereference) items to packages in the metapackage. If you set this property to `true`, you can add explicit references to just the packages you need.
```
